### ISSUE TITLE
`.Capabilities.kubeVersion` isn't available for Helm modules

### ISSUE THREAD

**Issue by jamesmeneghello** [garden-io/garden#2971]
### Current Behavior

`kubeVersion` isn't set when rendering Helm modules, which results in unexpected behaviour with some Helm charts. Most Bitnami charts set `ingressClassName` based on the `kubeVersion >1.19`, for example, which means creating any Bitnami chart with an ingress doesn't set `ingressClassName` at all.

### Expected behavior

Would expect the Kubernetes provider to pass the Kube version to `helm template --kube-version <version>`, either by querying the cluster to get the version or by allowing a version to be specified as part of the provider.

### Reproducible example

```yaml
kind: Module
type: helm
name: keycloak
repo: https://charts.bitnami.com/bitnami
chart: keycloak
version: "9.0.1"
values:
    ingress:
        enabled: true
        ingressClassName: nginx
```

### Workaround

Workaround for now (specifically with Bitnami charts) is to set the chart value `global.KubeVersion`, which it also checks. Any charts that rely only on `.Capabilities.KubeVersion` would not currently function as intended.

### Your environment

* **OS:** Ubuntu 20.04 LTS
* **How I'm running Kubernetes:** Kind, EKS

`garden version`: `12.40.0`

**Comment by thsig** [comment]
Makes sense! This should be a simple matter of adding a `kubeVersion` config field to the `helm` module type, and passing that to the relevant `helm` CLI invocations.

This would actually be a great first feature for anyone who's interested in having a go at it : )

**Comment by 10ko** [comment]
This should be fixed in `0.13.x`. Closing this issue.
